README.md

```diff
@@ -33,7 +33,13 @@ git clone --recursive https://github.com/openmultiplayer/open.mp
 git clone --recursive git@github.com:openmultiplayer/open.mp
 ```
 
-Note the use of the `--recursive` argument, because this repository contains submodules.
+Note the use of the `--recursive` argument, because this repository contains submodules.  We recommend you use `https:`, but configure git to redirect with:
+
+```
+git config --global url."git@github.com:".insteadOf "https://github.com/"
```

Comment:
Is there a reason we are recommending this to others?

Reply:
It used to be an issue when submodules were private but that's no longer the case